ospf6d/ospf6_flood.c

```diff
@@ -842,18 +842,6 @@ void ospf6_receive_lsa(struct ospf6_neighbor *from,
 				zlog_debug("Received is duplicated LSA");
 			SET_FLAG(new->flag, OSPF6_LSA_DUPLICATE);
 		}
```

Comment:
Minor nit: Please add logs to indicate received LSA is self originated and max-age.